style.css

```diff
@@ -219,25 +231,18 @@ body {
     min-height: 1080px;
     background-image: url(bg-images/4.jpg);
     background-repeat: no-repeat;
+    background-size: cover;
 }
 
-.buttons {
-    position: relative;
-    top: 60px;
-    right: 280px;
-}
-
-.button__back {
-    position: relative;
+.button {
     width: 75px;
     height: 75px;
     border: 1px solid gray;
     background-color: #FFFFFF;
     border-radius: 50%;
-    margin-bottom: -155px;
 }
 
-.button__back:before {
+.button:nth-child(1):before {
```

Comment:
Please create a separate class for button icon OR put it in the html.
For example: `button_back` and use it in html markup like `<button class="button button_back"></button>`